### PR TITLE
hotfix/Fix event server mongo uri

### PR DESCRIPTION
**Pull request by tokejepsen**
- collection name in uri creates a bad database name error
- uri was not composed correctly

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
Should go to `master` branch

File: pype/modules/ftrack/ftrack_server/lib.py [hunk 1]
```diff
@@ -51,9 +51,8 @@ def get_ftrack_event_mongo_info():
 
     if not _used_ftrack_url or components["database"] is None:
         components["database"] = database_name
-    components["collection"] = collection_name
```
Comment (iLLiCiTiT) [inline pype/modules/ftrack/ftrack_server/lib.py:54]: This line is required (for cases that `AVALON_MONGO` has collection in).

If connection won't work with `collection` in uri then there should be `components.pop("collection", None)` to be sure.

**Comment by tokejepsen** [comment]
> Should go to `master` branch

Its needed in develop though as well.

File: pype/modules/ftrack/ftrack_server/lib.py (hunk 1 above)
Comment (tokejepsen) [reply pype/modules/ftrack/ftrack_server/lib.py:54]: Good point. Done.

**Comment by iLLiCiTiT** [comment]
> Its needed in develop though as well.

Yes, but master is at first place, it should be hotfix because it's production breaking bug. These bugs should always be in master first (and with @mkolar git app it's automatically merged to develop too if branch name starts with `hotfix/***`)

**Pull request merged**